src/docker-osx-dev

```diff
@@ -389,7 +389,8 @@ function init_boot2docker {
 # Inspect especific configuration about the docker-machine
 #
 function inspect_docker_machine {
-  docker-machine inspect --format="$*" "$DOCKER_MACHINE_NAME" 2>&1
+  local result=$(docker-machine inspect --format="$*" "$DOCKER_MACHINE_NAME" 2>&1)
+  echo ${result/<no value>/}
```

Comment:
Perhaps a safer and easier-to-read approach is to have the `inspect_docker_machine` function `echo` the value if one is found or return -1 if it is not. You could then call it like this:

```
DOCKER_MACHINE_STORE_PATH=$(inspect_docker_machine "{{.StorePath}}" || inspect_docker_machine {{.AuthOptions.StorePath}})
```